modulemd-tools.spec

```diff
@@ -8,12 +8,12 @@ BuildArch: noarch
 URL: https://github.com/rpm-software-management/modulemd-tools
 Source0: https://github.com/rpm-software-management/modulemd-tools/archive/%{version}/%{name}-%{version}.tar.gz
 
-BuildRequires: libmodulemd >= 2
 BuildRequires: createrepo_c
 BuildRequires: argparse-manpage
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 BuildRequires: python3-gobject
+BuildRequires: python3-libmodulemd
```

Comment:
You probably want to set a minimum version here, if only to ensure you have all the features you are relying upon.

Reply:
Good thinking, will do

Reply:
Done, I went for `>= 2.9.3` instead of the latest one because I know everything works fine with that version, we don't require the last one.